hs_sitemap/sitemaps.py

```diff
@@ -1,13 +1,52 @@
 from django.contrib import sitemaps
 from django.urls import reverse
+from hs_core.models import BaseResource
+from hs_access_control.models import GroupAccess, Community
+from django.db.models import Q
 
 
 class PagesSitemap(sitemaps.Sitemap):
     priority = 0.5
-    changefreq = 'daily'
+    changefreq = 'weekly'
 
     def items(self):
         return ['home', 'login', 'apps']
 
     def location(self, item):
         return reverse(item)
+
+
+class CommunitiesSitemap(sitemaps.Sitemap):
+    priority = 0.5
+    changefreq = 'weekly'
+
+    def items(self):
+        return Community.objects.filter(active=True)
+
+    def location(self, item):
+        return f'/community/{ item.id }'
+
+
+class GroupsSitemap(sitemaps.Sitemap):
+    priority = 0.5
+    changefreq = 'daily'
+
+    def items(self):
+        return GroupAccess.objects.filter(active=True)
```

Comment:
I think this should be `return Group.objects.filter(gaccess__active=True)`

Reply:
Okay didn't seem to matter to me due to, but maybe there's a reason I'm not seeing?
https://github.com/hydroshare/hydroshare/blob/5125-sitemap/hs_access_control/models/group.py#L42-L53
```
class GroupAccess(models.Model):
...
    group = models.OneToOneField(Group, on_delete=models.CASCADE,
                                 editable=False,
                                 null=False,
                                 related_name='gaccess',
                                 related_query_name='gaccess',
                                 help_text='group object that this object protects')
```

Reply:
Normally, I would assume group id and gaccess id to match. When testing locally it seems to be different.

Reply:
ah interesting!